packages/react-native-media-player/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./lib",
+    "sourceMap": true,
+    "noImplicitAny": true,
+    "module": "commonjs",
+    "target": "es6",
+    "jsx": "react",
+    "declaration": true,
+    "allowJs": true,
+    "noEmit": false
+  },
+  "include": ["./src"],
+}
```

Comment:
Codacy found an issue: [Unexpected character ('}' (code 125)): was expecting double-quote to start field name](https://app.codacy.com/manual/yajananrao/Nadha/pullRequest?prid=5728824)